### DISCUSSION TITLE
[Docker-Compose] Setting specific storage location?

### DISCUSSION THREAD

**Discussion by dm7500** [ente-io/ente#3552] [Q&A]
Forgive me if I'm missing something obvious with this question, but if I want to run Ente and the rest of the docker stack from server A, but use a SMB share from server B for storage (which is mounted on server A), what service would I change the volume info on? Museum, Postgres, or Minio? I'm not running any backups or replication, just need to test out basic functionality. 

I'm fairly well versed in Docker, but I think I'm a bit tripped up by the architecture here.

**Comment by mnvr** [comment]
>  use SMB share from server B for storage... what service would I change the volume info on?

Minio. That's the part which stores the actual data blobs.
